queries/ecma/indents.scm

```diff
@@ -75,3 +76,15 @@
   (comment)
   (ERROR)
 ] @indent.auto
+
+(if_statement
+  consequence: (_) @indent.dedent
+  (#not-kind-eq? @indent.dedent statement_block)) @indent.begin
+
+(while_statement
+  body: (_) @indent.dedent
+  (#not-kind-eq? @indent.dedent statement_block)) @indent.begin
+
+(for_statement
+  body: (_) @indent.dedent
+  (#not-kind-eq? @indent.dedent statement_block)) @indent.begin
```

Comment:
I personally think this doesn't happen frequent enough in modern javascript land to really need to care about. Especially when most formatters and autocompletion when doing `foo.for` will result in `for (const bar in foo) {}`

Reply:
Sounds good: same for while statement?

Reply:
I'd say so.